autode/utils.py

```diff
@@ -21,10 +24,12 @@
     CouldNotGetProperty,
 )
 
-try:
-    mp.set_start_method("fork")
-except RuntimeError:
-    logger.warning("Multiprocessing context has already been defined")
+#try:
+#    mp.set_start_method("fork")
+#except RuntimeError:
+#    logger.warning("Multiprocessing context has already been defined")
+#except ValueError:
+#    logger.warning("Multiprocessing is not used in Windows")
```

Comment:
This needs to be either (a) removed if it's not required and joblib handles the case when you can't fork the process or (b) fixed!